cpp/lazperf/vlr.hpp

```diff
@@ -113,12 +127,50 @@ struct eb_vlr : public vlr
 
     std::vector<ebfield> items;
 
-    LAZPERF_EXPORT eb_vlr(size_t bytes);
+    eb_vlr();
+    eb_vlr(int ebCount);
+    virtual ~eb_vlr();
 
-    LAZPERF_EXPORT virtual size_t size() const;
-    LAZPERF_EXPORT virtual std::vector<char> data() const;
-    LAZPERF_EXPORT virtual vlr_header header() const;
-    LAZPERF_EXPORT void addField();
+    static eb_vlr create(std::istream& in, int byteSize);
+    void read(std::istream& in, int byteSize);
+    void write(std::ostream& out) const;
+    virtual size_t size() const;
+    virtual vlr_header header() const;
+    void addField();
+};
+
+struct LAZPERF_EXPORT wkt_vlr : public vlr
+{
+public:
+    std::string wkt;
+
+    wkt_vlr();
+    wkt_vlr(const std::string& s);
+    virtual ~wkt_vlr();
+
+    static wkt_vlr create(std::istream& in, int byteSize);
+    void read(std::istream& in, int byteSize);
+    void write(std::ostream& out) const;
+    virtual size_t size() const;
+    virtual vlr_header header() const;
+};
+
+struct LAZPERF_EXPORT copc_vlr : public vlr
```

Comment:
Does this belong here? I don't have a strong opinion, but I thought we stated that COPC library convenience wasn't going to be implemented in laz-perf.

Reply:
It simplifies my life, so...